web/angular-wallet/src/app/main/set-reward-recipient/set-reward-recipient.component.html

```diff
@@ -0,0 +1,49 @@
+<div class="page-layout blank simple" fusePerfectScrollbar *ngIf="fees">
+
+  <div class="header accent p-24 h-160" fxLayout="column" fxLayoutAlign="center center" fxLayout.gt-xs="row"
+      fxLayoutAlign.gt-xs="space-between center">
+
+      <div fxLayout="column" fxLayoutAlign="center center" fxLayout.gt-xs="column" fxLayoutAlign.gt-xs="center start">
+          <div fxLayout="row" fxLayoutAlign="start center">
+              <mat-icon class="secondary-text s-18" routerLink="/dashboard">home</mat-icon>
+              <mat-icon class="secondary-text s-16">chevron_right</mat-icon>
+              <span class="secondary-text">{{ 'set_reward_recipient' | i18n }}</span>
+          </div>
+          <div class="h1 mt-16">{{ 'set_reward_recipient' | i18n }}</div>
+      </div>
+  </div>
+
+  <div class="p-24 content">
+
+    <form (ngSubmit)="onSubmit($event);" #setRewardRecipientForm="ngForm">
+
+      <burst-recipient-input appearance="outline" (recipientChange)="onRecipientChange($event)"></burst-recipient-input>
```

Comment:
I prefer the floating-label version (no outline)... we should be consistent with our input fields.